libraries/AP_RangeFinder/AP_RangeFinder_Benewake_TFMiniPlus.cpp

```diff
@@ -102,6 +104,8 @@ bool AP_RangeFinder_Benewake_TFMiniPlus::init()
     if (val[5] * 10000 + val[4] * 100 + val[3] < 20003) {
         hal.console->printf(DRIVER ": minimum required FW version 2.0.3, but version %u.%u.%u found\n",
                             val[5], val[4], val[3]);
+        gcs().send_text(MAV_SEVERITY_INFO, ": minimum required FW version 2.0.3, but version %u.%u.%u found\n",
```

Comment:
Please remove the extra line-feed here.  It usually leads to the GCS inserting extra vertical whitespace where none is required.